$OpenBSD: patch-autoconf_inc_m4,v 1.2 2010/11/02 13:10:51 dcoppa Exp $
Don't use evil ln -s between lib.so.x.y and lib.so.x and lib.so
--- autoconf_inc.m4.orig	Wed Oct 27 12:30:11 2010
+++ autoconf_inc.m4	Wed Oct 27 12:32:43 2010
@@ -275,7 +275,7 @@ dnl ### begin block 20_COND_OFFICIAL_BUILD_1_PLATFORM_
 dnl ### begin block 20_COND_PLATFORM_MACOSX_0_USE_SOVERCYGWIN_0_USE_SOVERSION_1[../../contrib/build/deprecated/deprecated.bkl,../../contrib/build/fl/fl.bkl,../../contrib/build/foldbar/foldbar.bkl,../../contrib/build/gizmos/gizmos.bkl,../../contrib/build/mmedia/mmedia.bkl,../../contrib/build/net/net.bkl,../../contrib/build/ogl/ogl.bkl,../../contrib/build/plot/plot.bkl,../../contrib/build/stc/stc.bkl,../../contrib/build/svg/svg.bkl,wx.bkl] ###
     COND_PLATFORM_MACOSX_0_USE_SOVERCYGWIN_0_USE_SOVERSION_1="#"
     if test "x$PLATFORM_MACOSX" = "x0" -a "x$USE_SOVERCYGWIN" = "x0" -a "x$USE_SOVERSION" = "x1" ; then
-        COND_PLATFORM_MACOSX_0_USE_SOVERCYGWIN_0_USE_SOVERSION_1=""
+        COND_PLATFORM_MACOSX_0_USE_SOVERCYGWIN_0_USE_SOVERSION_1="#"
     fi
     AC_SUBST(COND_PLATFORM_MACOSX_0_USE_SOVERCYGWIN_0_USE_SOVERSION_1)
 dnl ### begin block 20_COND_PLATFORM_MACOSX_0_USE_SOVERSION_1[../../contrib/build/deprecated/deprecated.bkl,../../contrib/build/fl/fl.bkl,../../contrib/build/foldbar/foldbar.bkl,../../contrib/build/gizmos/gizmos.bkl,../../contrib/build/mmedia/mmedia.bkl,../../contrib/build/net/net.bkl,../../contrib/build/ogl/ogl.bkl,../../contrib/build/plot/plot.bkl,../../contrib/build/stc/stc.bkl,../../contrib/build/svg/svg.bkl,wx.bkl] ###
@@ -857,7 +857,7 @@ dnl ### begin block 20_COND_USE_RTTI_1[../../contrib/b
 dnl ### begin block 20_COND_USE_SOTWOSYMLINKS_1[../../contrib/build/deprecated/deprecated.bkl,../../contrib/build/fl/fl.bkl,../../contrib/build/foldbar/foldbar.bkl,../../contrib/build/gizmos/gizmos.bkl,../../contrib/build/mmedia/mmedia.bkl,../../contrib/build/net/net.bkl,../../contrib/build/ogl/ogl.bkl,../../contrib/build/plot/plot.bkl,../../contrib/build/stc/stc.bkl,../../contrib/build/svg/svg.bkl,wx.bkl] ###
     COND_USE_SOTWOSYMLINKS_1="#"
     if test "x$USE_SOTWOSYMLINKS" = "x1" ; then
-        COND_USE_SOTWOSYMLINKS_1=""
+        COND_USE_SOTWOSYMLINKS_1="#"
     fi
     AC_SUBST(COND_USE_SOTWOSYMLINKS_1)
 dnl ### begin block 20_COND_USE_SOVERCYGWIN_1_USE_SOVERSION_1[../../contrib/build/deprecated/deprecated.bkl,../../contrib/build/fl/fl.bkl,../../contrib/build/foldbar/foldbar.bkl,../../contrib/build/gizmos/gizmos.bkl,../../contrib/build/mmedia/mmedia.bkl,../../contrib/build/net/net.bkl,../../contrib/build/ogl/ogl.bkl,../../contrib/build/plot/plot.bkl,../../contrib/build/stc/stc.bkl,../../contrib/build/svg/svg.bkl,wx.bkl] ###
@@ -873,7 +873,7 @@ dnl ### begin block 20_COND_USE_SOVERLINUX_1[../../con
     fi
     AC_SUBST(COND_USE_SOVERLINUX_1)
 dnl ### begin block 20_COND_USE_SOVERSION_0[../../contrib/build/deprecated/deprecated.bkl,../../contrib/build/fl/fl.bkl,../../contrib/build/foldbar/foldbar.bkl,../../contrib/build/gizmos/gizmos.bkl,../../contrib/build/mmedia/mmedia.bkl,../../contrib/build/net/net.bkl,../../contrib/build/ogl/ogl.bkl,../../contrib/build/plot/plot.bkl,../../contrib/build/stc/stc.bkl,../../contrib/build/svg/svg.bkl,wx.bkl] ###
-    COND_USE_SOVERSION_0="#"
+    COND_USE_SOVERSION_0=""
     if test "x$USE_SOVERSION" = "x0" ; then
         COND_USE_SOVERSION_0=""
     fi
